Mod2/Lessons/Week3/SQLAggregates.md

```diff
@@ -0,0 +1,158 @@
+# SQL Aggregates
+
+## Learning Goals
+* Develop a visualization for grouping and aggregating data
+* Use GROUP BY, COUNT, SUM, and AVERAGE
+
+** Instructor Note ** If students need a totally fresh start on their database, you can give them the script below:
+```sql
+DROP TABLE songs, artists;
+
+CREATE TABLE artists (
+	id serial PRIMARY KEY,
+	name varchar(255),
+	hometown varchar(255),
+	years_active varchar(255)
+);
+
+
+CREATE TABLE songs (
+	id serial PRIMARY KEY,
+	title varchar(50),
+	length int,
+	play_count int,
+    artist_id int REFERENCES artists
+);
+```
+
+## Warm Up
+
+For this lesson, you will want to have a good amount of data in your Set List database.  Use the script below to insert songs and artists into your database:
+
+```
+INSERT INTO artists ( name, hometown, years_active )
```

Comment:
Thoughts on adding genre to our artist table? I like how you use it in the video and I think it allows for some helpful examples.

Reply:
I like it, let's add it :)